community/cypher/compatibility-suite/src/test/resources/cypher/MergeAcceptance.feature

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2002-2015 "Neo Technology,"
+# Network Engine for Objects in Lund AB [http://neotechnology.com]
+#
+# This file is part of Neo4j.
+#
+# Neo4j is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+Feature: MergeAcceptanceTest
+
+  Scenario: identifiers of deleted nodes should not be able to cause errors in later merge actions that do not refer to them
+    Given init: CREATE (:A)-[:R]->(:B)-[:R]->(:C)
+    When running: MATCH (a:A) -[ab]-> (b:B) -[bc]-> (c:C) DELETE ab, bc, b, c MERGE (newB:B) MERGE (a) -[:REL]->  (newB) MERGE (newC:C) MERGE (newB) -[:REL]-> (newC);
```

Comment:
Could we agree on a common style of how we write queries in the acceptance suite? I think capital keywords, and no spaces around arrows is the blessed way that we use in the manual.